src/components/map/map.e2e.ts

```diff
@@ -13,6 +12,7 @@ describe("gx-map", () => {
     await page.waitForChanges();
     element = await page.find("gx-map");
     const innerContent = await element.find("div .leaflet-control-container");
+    await page.waitForChanges();
```

Comment:
No changes were made, so I'm not sure what this line is doing.

Reply:
You are right! Perhaps in one of my tries to add a test for this feature I add that line and forgot to delete...
I will delete it in the next commit.